Running the server from docker-compose

### Describe the bug
The build fails with the error:
```
...
artemis-server_1  | During the build, one or more dependencies that were declared without a version failed to resolve:
artemis-server_1  |     org.springframework.boot:spring-boot-devtools:
artemis-server_1  |
artemis-server_1  | Did you forget to apply the io.spring.dependency-management plugin to the Artemis project?
artemis-server_1  |
artemis-server_1  |
artemis-server_1  | Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
artemis-server_1  | Use '--warning-mode all' to show the individual deprecation warnings.
artemis-server_1  | See https://docs.gradle.org/6.6/userguide/command_line_interface.html#sec:command_line_warnings
artemis-server_1  |
artemis-server_1  | FAILURE: Build failed with an exception.
artemis-server_1  |
artemis-server_1  | * What went wrong:
artemis-server_1  | Execution failed for task ':bootJar'.
artemis-server_1  | > Could not resolve all files for configuration ':developmentOnly'.
artemis-server_1  |    > Could not find org.springframework.boot:spring-boot-devtools:.
artemis-server_1  |      Required by:
artemis-server_1  |          project :
...
```
#### To Reproduce
Setup development setup using docker-compose.

#### Expected behavior
Application launches

### Environment
<!-- Please complete the following information: -->
<details><pre>
 - OS: Ubuntu 20.04
</pre></details>

#### Additional context
The error goes way when I add a version to the version to the `org.springframework.boot:spring-boot-devtools` dependency. in profile_dev.gradle

--- a/gradle/profile_dev.gradle
+++ b/gradle/profile_dev.gradle
@@ -6,7 +6,7 @@ configurations {
 }
 
 dependencies {
-    developmentOnly "org.springframework.boot:spring-boot-devtools"
+    developmentOnly "org.springframework.boot:spring-boot-devtools:${spring_boot_version}"
 }
 
 def profiles = 'dev'

